runtime/lua/lsp/util.lua

```diff
@@ -1,3 +1,5 @@
+local util = require('neovim.util')
```

Comment:
Use `nvim` for technical identifiers (basically everywhere except "marketing" material).

However I don't think we're planning to have a separate `nvim` object in Lua. There's already a `vim` object, and that's where we should put our Lua stdlib IIRC. cc @KillTheMule

Reply:
I've made a distinction between `helpers` and `lua_api`, both contained in `vim`, [see here](https://github.com/neovim/neovim/pull/8677/files#diff-f2670a0ec9153855371c044ecddf842b) for the rough first shot.

I agree with @justinmk, just go ahead and we'll reorganize things later (I'm very short on time these days, this will have to wait anyways).

Reply:
Sounds good on both fronts. I can always just do `local util = vim.helpers` or something like that when we're further along on both fronts.

Reply:
@tjdevries ^